src/components/Identity/index.js

```diff
@@ -88,11 +88,19 @@ const createIdentity = ({ config, logger, cookie }) => {
           const timestamp = parseInt(cookie.get(ID_SYNC_TIMESTAMP) || 0, 36);
 
           if (config.idSyncsEnabled && nowInHours > timestamp) {
-            event.mergeQuery({
+            const identityQuery = {
               identity: {
                 exchange: true
               }
-            });
+            };
+
+            const containerId = parseInt(config.idSyncContainerId, 10);
```

Comment:
I feel it would be better to add config validator that rejects the user's value if it's not a number.